test/test_markdown.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import warnings
+import json
+import pytest
+from jsonschema import validate
+import commonmark
```

Comment:
nitpick: usually stdlib imports are separated by an empty line from the external imports